Makefile

```diff
@@ -1,9 +1,8 @@
 SOURCES := $(wildcard src/*.sv)
-all:
-
+LATEX := lualatex
 COMPILER ?= iverilog
 
-all: out/cpu
+all: out/cpu out/instructionset.pdf
```

Comment:
I think it would be better to allow to compile it separately so LaTeX is not a depedency of the project

Reply:
d'ailleurs, `all` devrait dépendre de `build` et non `out/cpu` (ou au moins, mettre `out/cpu` dans une variable et utiliser cette variable partout au lieu de hardcoder `out/cpu`)